graphql_compiler/tests/schema_transformation_tests/test_rename_schema.py

```diff
@@ -14,11 +15,12 @@
 class TestRenameSchema(unittest.TestCase):
     def test_rename_visitor_type_coverage(self):
         """Check that all types are covered without overlap."""
-        all_types = set(ast_type.__name__ for ast_type in QUERY_DOCUMENT_KEYS)
+        all_types_snake_case = set(ast_type for ast_type in QUERY_DOCUMENT_KEYS)
```

Comment:
What is the type of `QUERY_DOCUMENT_KEYS`? This should probably be `set(QUERY_DOCUMENT_KEYS)` or maybe even just `QUERY_DOCUMENT_KEYS` used directly.